data/strings/en.ini

```diff
@@ -62,6 +62,7 @@ deleting_tilemaps_will_delete_tilesets = Warning\n<<Deleting the following tilem
 cannot_file_overwrite_on_export = Overwrite Warning\n<<You cannot Export with the same name (overwrite the original file).\n<<Use "File > Save As" menu option in that case.\n||&OK
 cannot_open_file = Problem<<Cannot open file:<<{0}||&OK
 cannot_open_folder = Problem<<Cannot open folder:<<{0}||&OK
+cannot_open_theme = Error<<The selected theme ID is "default". Please change<<the theme ID inside "package.json" to make it work.||&OK
```

Comment:
Is this message needed? Because it looks like in the place where `cannot_open_theme` is used it should be `cannot_install_default_extension` instead (e.g. when we use "Add Extension" we could show `cannot_install_default_extension`).